src/Auth.hpp

```diff
@@ -14,34 +14,43 @@
 #include "ConfigVar.hpp"
 #include "HTTPHeader.hpp"
 #include "ListContainer.hpp"
+#include "LOptionContainer.hpp"
 
 // DEFINES
 
 // success
-#define DGAUTH_OK 0
+#define E2AUTH_OK 0
 
 // auth info required for this method not found (continue querying other plugins)
-#define DGAUTH_NOMATCH 2
+#define E2AUTH_NOMATCH 2
 
 // auth info found, but no such user in filtergroupslist (stop querying plugins - use this code with caution!)
-#define DGAUTH_NOUSER 3
+#define E2AUTH_NOUSER 3
+
+// auth info found, but no such user in group for this plugin  stop querying plugins - use this code with caution!)
+#define E2AUTH_NOGROUP 3
 
 // redirect the user to a login page
-#define DGAUTH_REDIRECT 4
+#define E2AUTH_REDIRECT 4
 
 // allow access to http[s] based auth system, but with no persitance
-#define DGAUTH_OK_NOPERSIST 9
+#define E2AUTH_OK_NOPERSIST 9
 
 // auth plugin found a partial or incomplet answer (Eg NTLM): just break ident loop for this request  
-#define DGAUTH_NOIDENTPART 5
+#define E2AUTH_NOIDENTPART 5
 
 // any < 0 return code signifies error
 
-// auth_rec structure for use by storyboarding
+// auth_rec structure for use by storyboarding/extended logging
 struct auth_rec {
     String user_name;
     bool is_authed;
     int filter_group;
+    bool is_proxy = false;
+    bool is_transparent = false;
+    bool is_icap = false;
```

Comment:
Codacy found an issue: [struct member 'auth_rec::is_icap' is never used.](https://app.codacy.com/manual/numsys/e2guardian/pullRequest?prid=6238852)